R/Attributes.R

```diff
@@ -62,7 +63,12 @@ sourceCpp <- function(file = "",
         if (grepl(' ', basename(file), fixed=TRUE)) {
             stop("The filename '", basename(file), "' contains spaces. This ",
                  "is not permitted.")
-        }                                                       # #nocov end
+        }
+    } else {
+        if (verbose && windowsDebugDLL) {
+            message("The 'windowsDebugDLL' toggle is ignored on non-Windows platforms.")
+            windowsDebugDLL <- FALSE    # now we do not need to deal with OS choice below
```

Comment:
Shouldn't this be outside the if()? As it stands now, this will only trigger if `verbose = TRUE` is passed (it normally isn't.)

Reply:
Nice catch.  I think you are correct.  Will fix.

Reply:
New variant:

```R
        if (windowsDebugDLL) {                                                                                                                                                                                     
            if (verbose) message("The 'windowsDebugDLL' toggle is ignored on non-Windows platforms.")                                                                                                              
            windowsDebugDLL <- FALSE    # now we do not need to deal with OS choice below                                                                                                                          
        }                                                       # #nocov end                                                                                                                                       
    }       
```

Better...